manager/integration/tests/test_ha.py

```diff
@@ -2620,107 +2620,145 @@ def test_replica_failure_during_attaching(settings_reset, client, core_api, volu
 
 
 @pytest.mark.skip(reason="TODO") # NOQA
-def test_engine_image_missing_on_some_nodes():
+def test_engine_image_miss_scheduled_perform_volume_operations():
     """
-    This e2e test follows the manual test steps at:
-    https://github.com/longhorn/longhorn/issues/2081#issuecomment-783747541
-
-    Steps:
-
-    Preparation:
-    1. Set up a backup store
-    2. Let's name the 3 nodes: node-1, node-2, node-3
-
-    Case 1: Test volume operations when engine image DaemonSet is miss
+    Test volume operations when engine image DaemonSet is miss
     scheduled
+
     1. Create a volume, vol-1, of 3 replicas
-    2. Create another volume, vol-2, of 3 replicas
-    3. Taint node-1 with the taint: key=value:NoSchedule
-    4. Verify that we can attach, take snapshot, take a backup,
+    2. Taint node-1 with the taint: key=value:NoSchedule
+    3. Verify that we can attach, take snapshot, take a backup,
        expand, then detach vol-1
+    """
+    pass
 
-    Case 2: Test volume operations when engine image DaemonSet is not fully
+
+@pytest.mark.skip(reason="TODO") # NOQA
+def test_engine_image_not_fully_deployed_perform_volume_operations():
+    """
+    Test volume operations when engine image DaemonSet is not fully
     deployed
-    1. Continue from case #1
-    2. Attach vol-1 to node-1. Change the number of replicas of vol-1
+
+    1. Create 2 volumes, vol-1 and vol-2 with 3 replicas
+    2. Taint node-1 with the taint: key=value:NoSchedule
+    3. Attach vol-1 to node-1. Change the number of replicas of vol-1
        to 2. Delete the replica on node-1
-    3. Delete the pod on node-1 of the engine image DaemonSet.
+    4. Delete the pod on node-1 of the engine image DaemonSet.
        Or delete the engine image DaemonSet and wait for Longhorn
        to automatically recreates it.
```

Comment:
Can we make these steps as prerequisite for the tests `test_engine_image_not_fully_deployed_perform_volume_operations` and `test_engine_image_not_fully_deployed_perform_engine_upgrade`